### DISCUSSION TITLE
Should files be committed into the target/ directory?

### DISCUSSION THREAD

**Discussion by DL8** [amichai-bd/riscv-multi-core-lotr#70] [Q&A]
If I understand correctly, target/ is where execution logs will be written. Is there any reason to commit files there?

**Comment by amichai-bd** [comment]
Target should be part of ".gitignor" and be not committed.
We did not enforce it at first, due to problems with running the tests - and we wanted to review the results.
I will Talk to the Designers and make sure they remove the "trks" from the target, add it to the ignore, and out a "README" where they describe how to run the tests and generate the target trks.

Note: we are discussing a big change in the way we run tests and logs - so this will be part of that task.

**Comment by DL8** [comment]
Regarding ignoring target/, personally I prefer putting a strict
".gitignore" inside the directory (ignore everything except itself, see in
the library's outputs). It guatantees that the directory will exist,
instead of having to recreate it every time.
Regarding readme, it can be somewhere in the repo. I don't see a reason why
generate it as a build output.
